lib/amplitude_web.dart

```diff
@@ -27,6 +29,30 @@ class AmplitudeFlutterPlugin {
   /// Note: Check the "federated" architecture for a new way of doing this:
   /// https://flutter.dev/go/federated-plugins
   Future<dynamic> handleMethodCall(MethodCall call) async {
+    if (call.method == 'init') {
+      var args = call.arguments;
+      String apiKey = args['apiKey'];
+      JSObject configuration = getConfiguration(call);
+
+      // Set library
+      Amplitude instance = amplitude.createInstance();
+      instance.add(createJSInteropWrapper(FlutterLibraryPlugin(
+          args['library'] ?? 'amplitude_flutter/unknown')));
+      instance.init(apiKey, configuration);
+
+      instances[args['instanceName'] ?? '\$default_instance'] = instance;
```

Comment:
nit: use `Constants.defaultInstanceName`

Reply:
Fixed! Sorry, I'm not very familiar with the project 🙄